fsw/src/crypto.c

```diff
@@ -566,7 +566,8 @@ static int32 Crypto_SA_config(void)
 
     // Security Associations
         // SA 1 - CLEAR MODE
-        sa[1].sa_state = SA_OPERATIONAL;
+        // sa[1].sa_state = SA_OPERATIONAL;
+        sa[1].sa_state = SA_NONE;
```

Comment:
I think we should leave the existing SA's as is whenever possible to try and preserve previous test data being used as unit tests against this code. Whenever you need an SA in a state that isn't already laid out here, we should just add a new SA rather than change an existing one.

Reply:
Probably not a bad idea. For some testing we need to turn on/off SAs depending on the data we're using. for now can either just manually set them to on/off during a test case after calling cryptolib_init, in the future likely interaction with SADB API. Resolved in upcoming PR.